index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Ask It</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="styles/common.css">
+		<link rel="stylesheet" href="styles/header/header.css">
+		<link rel="stylesheet" href="styles/landing/landing.css">
+		<link rel="stylesheet" href="styles/guide/guide.css">
+	</head>
+	<body>
+		<header class="header">
+			<div class="header__logo-container">
+				<img class="header__logo-image" src="shared/header-logo.png">
```

Comment:
please provide `alt = ...` attribute for image, so that user will see some text in case the image wasn't loaded